test/common/packagelib.py

```diff
@@ -63,7 +63,8 @@ def setUp(self):
             self.restore_dir("/etc/apt", reboot_safe=True)
             self.machine.execute("echo > /etc/apt/sources.list; rm -f /etc/apt/sources.list.d/*; apt-get clean; apt-get update")
         elif self.backend == "alpm":
-            self.restore_dir("/var/lib/pacman", reboot_safe=True)
+            self.restore_dir("/var/lib/pacman/local", reboot_safe=True)
+            self.restore_dir("/var/lib/pacman/sync", reboot_safe=True)
```

Comment:
This feels brittle to me -- if there ever is, or will be (future version) another file/dir, this goes out of sync.